packages/client/vite.config.mjs

```diff
@@ -67,6 +67,10 @@ export default defineConfig(({ mode }) => {
           find: "constants",
           replacement: path.resolve("./src/constants"),
         },
+        {
+          find: "@/constants",
+          replacement: path.resolve("./src/constants"),
+        },
```

Comment:
You can replace this with the catch-all `"@": path.resolve(__dirname, "src")`, as long as you put it after the `@budibase/xxx` aliases below.

Reply:
I did try that initially, but getting the following error:
![image](https://github.com/user-attachments/assets/9651711c-1161-4bf1-b406-bd72a1c0875e)

Reply:
Very interesting. I wonder what the difference is between this and the builder package, since we use that there.